src/github/dto/update.practices.dto.ts

```diff
@@ -0,0 +1,11 @@
+import { IsNotEmpty } from 'class-validator';
+import { ApiProperty } from '@nestjs/swagger';
+
+export class UpdatePracticesDto {
+  @ApiProperty({
+    description: 'Practices can be strings or a single paragraph.',
+    example: ['practice1', 'practice2'],
+  })
+  @IsNotEmpty()
+  practices: string[];
```

Comment:
PullProofAI

The `UpdatePracticesDto` class is missing validation for the `practices` field. Adding validation ensures that the `practices` field is not empty.
```diff
+   @IsNotEmpty()
```


<!-- This is an auto-generated comment by PullProofAI -->